src/gsi/setupspd.f90

```diff
@@ -528,7 +528,7 @@ subroutine setupspd(obsLL,odiagLL,lunin,mype,bwork,awork,nele,nobs,is,conv_diags
      ddiff = spdob-spdges
      
      if (aircraft_recon) then
-       if ( nty == 292 ) then 
+       if ( nty == 213 ) then 
          ratio_errors=error/(uv_doe_a_292*abs(ddiff)+uv_doe_b_292)
```

Comment:
Same comment as for gsimod.F90 and obsmod.F90.   Please replace uv_doe_a_292 with uv_doe_a_213.   Same for uv_doe_b_292.  It should be uv_doe_b_213.    This improves readability and consistency of code.

Reply:
Updated. Pushed changes to the branch.